src/components/Search/SelectedRevisionItem.tsx

```diff
@@ -29,46 +27,61 @@ import {
 const base = Strings.components.searchDefault.base;
 const warning = base.collapsed.warnings.comparison;
 
+interface InProgressState {
+  revs: RevisionsList[];
+  repos: Repository['name'][];
+  isInProgress: boolean;
+}
+
 interface SelectedRevisionItemProps {
   index: number;
   item: RevisionsList;
-  mode: ThemeMode;
   repository: Repository['name'];
-  searchType: InputType;
-  isWarning?: boolean;
   formIsDisplayed: boolean;
   isEditable: boolean;
+  isBase: boolean;
+  mode: ThemeMode;
+  inProgress: InProgressState;
+  isWarning: boolean;
+  setInProgress: React.Dispatch<React.SetStateAction<InProgressState>>;
 }
 
 function SelectedRevisionItem({
   index,
   item,
-  mode,
   repository,
-  searchType,
-  isWarning,
   formIsDisplayed,
   isEditable,
+  isBase,
+  mode,
+  inProgress,
+  setInProgress,
+  isWarning,
 }: SelectedRevisionItemProps) {
+  const searchType = isBase ? 'base' : 'new';
   const styles = SelectRevsStyles(mode);
-  const revisionHash = truncateHash(item.revision);
+  const revisionHash = item ? truncateHash(item.revision) : '';
```

Comment:
Why this ternary, is it possible that item is absent?

Reply:
Yes, something before made the item absent, but that's all fixed so this irrelevant code.